cmd/watt/main.go

```diff
@@ -125,12 +130,14 @@ func runWatt(cmd *cobra.Command, args []string) {
 		for _, err := range errs {
 			log.Println(err)
 		}
-		os.Exit(1)
+		return 1
 	}
+
+	return 0
 }
 
 func main() {
 	if err := rootCmd.Execute(); err != nil {
-		log.Fatalln(err)
+		log.Println(err)
```

Comment:
Shouldn't you still `os.Exit(1)` in this case? I think the original might have been fine here.